README.md

```diff
@@ -94,38 +97,36 @@ Create the following configuration file in `workbench/settings_pb.py`:
 from settings import *
 
 INSTALLED_APPS += ('problem_builder',)
-DATABASES['default']['NAME'] = 'var/workbench.db'
 ```
 
-Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. `problem-builder` uses MySQL 5.6 and hence you can quickly spin up
-an instance of MySQL 5.6 with docker. Run the following command to start a MySQL 5.6 server.
+By default, the `xblock-sdk` uses the SQLite database but MySQL
+can be used by specifying an environment variable `WORKBENCH_DATABASES` in
+the following format.
 
 ```bash
-docker run --rm -it -p 3306:3306 -e MYSQL_ROOT_PASSWORD=rootpw -e MYSQL_DATABASE=db circleci/mysql:5.6 mysqld --character-set-server=latin1 --collation-server=latin1_swedish_ci
+export WORKBENCH_DATABASES='{"default": {"ENGINE": "django.db.backends.mysql", "NAME": "db", "USER": "root", "PASSWORD": "rootpw", "HOST": "127.0.0.1", "PORT": "3306"}}'
```

Comment:
The port is set to 3306 here but 3307 later.

Reply:
Thanks for the catch, I have updated the PORT :)